server/src/controllers/eventController.ts

```diff
@@ -85,35 +85,43 @@ eventController.post('/checkout', async (req: Request, res: Response) => {
       orderTicketItems,
       ticketTotal,
       eventInstanceQueries,
-    } = await getTicketItems(cartItems, prisma);
+    } = await getTicketItems(ticketCartItems, prisma);
+
+    const {
+      subscriptionCartRows,
+      orderSubscriptionItems,
+      subscriptionTotal,
+    } = await getSubscriptionItems(prisma, subscriptionCartItems);
 
     const {
       donationItem,
       donationCartRow,
       donationTotal,
     } = getDonationItem(donation);
 
-    const discountAmount = discount.code != ''? getDiscountAmount(discount, ticketTotal): 0;
+    const discountAmount = discount.code != ''? getDiscountAmount(discount, ticketTotal+subscriptionTotal): 0;
```

Comment:
Since a discount is only validated based on the Ticket contents of the cart. I imagine we'll also want to only get a discount amount based off the ticket total.

Reply:
Updated